sdk/python/lib/setup.py

```diff
@@ -17,13 +17,19 @@
 from setuptools import setup, find_packages
 
 
+VERSION = "3.0.0"
```

Comment:
I was going to default this to `"0.0.0"` like we do in our generated provider SDKs, but that was less useful in practice. I tend to want to use a `-e` editable install of the `pulumi` package in a sample app that is using other SDKs like `pulumi-aws`, which require `pulumi>=3.0.0,<4.0.0`.